suite-native/module-onboarding/src/screens/WelcomeScreen.tsx

```diff
@@ -25,49 +31,109 @@ const imageContainerStyle = prepareNativeStyle(() => ({
     top: 0,
 }));
 
+const contentStyle = prepareNativeStyle(utils => ({
+    width: '100%',
+    backgroundColor: utils.colors.backgroundAlertRedBold,
+    alignItems: 'center',
+}));
+
+const cardStyle = prepareNativeStyle(utils => ({
+    marginHorizontal: utils.spacings.medium,
+    marginTop: utils.spacings.medium,
+    padding: utils.spacings.large,
+    borderRadius: 20,
+    borderColor: utils.colors.borderSubtle,
+    flex: 1,
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    borderWidth: utils.borders.widths.small,
+}));
+
 export const WelcomeScreen = () => {
+    const { translate } = useTranslate();
+
+    const { isUsbDeviceConnectFeatureEnabled: isUsbConnectEnabled } =
+        useIsUsbDeviceConnectFeatureEnabled();
+
     const navigation =
         useNavigation<
             StackNavigationProps<OnboardingStackParamList, OnboardingStackRoutes.Welcome>
         >();
-    const { applyStyle } = useNativeStyles();
+
+    const { applyStyle, utils } = useNativeStyles();
+
     const colorScheme = useActiveColorScheme();
 
     const handleRedirect = () => {
-        navigation.navigate(OnboardingStackRoutes.TrackBalances);
+        navigation.navigate(
+            isUsbConnectEnabled
+                ? OnboardingStackRoutes.ConnectTrezor
+                : OnboardingStackRoutes.TrackBalances,
+        );
     };
 
-    const getImageSource = () => {
-        if (colorScheme === 'dark') {
+    const isDarkMode = colorScheme === 'dark';
+
+    const getImageSource = useMemo(() => {
+        if (isDarkMode) {
             return require('../assets/darkRectangles.png');
         }
         return require('../assets/rectangles.png');
-    };
+    }, [isDarkMode]);
 
     return (
         <>
-            <Screen isScrollable={false}>
-                <Box flex={1} justifyContent="space-between" alignItems="center">
+            <Box style={applyStyle(contentStyle)} />
+            <Box style={applyStyle(imageContainerStyle)}>
+                <Image source={getImageSource} resizeMode="contain" />
+            </Box>
```

Comment:
This should be handled with https://reactnative.dev/docs/imagebackground I think

Reply:
done in 829658806